checklist.py

```diff
@@ -144,6 +146,71 @@ def _system_call(self, cmd):
         return stdout, stderr, return_value
 
 
+class CopyrightHeadersValidator(RepoValidator):
+    """Flag library files with non-standard copyright headers
+
+    See the current standard for scikit-bio's copyright headers at
+    ``http://scikit-bio.org/docs/latest/development/new_module.html``
+
+    Parameters
+    ----------
+    skip_dirs : iterable of str, optional
+        Directory names to skip during validation. Defaults to skipping any
+        directories named ``'data'`` or ``'__pycache__'`` (and anything
+        contained within them).
+
+    """
+
+    reason = "Files non-conforming to standard copyright headers"
```

Comment:
It could be helpful to point to the docs that describe this format:

http://scikit-bio.org/docs/latest/development/new_module.html